setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='mfl',
+    version='2.0.0',
+    description='Master falicity list',
+    long_description=(''),
```

Comment:
Write a `README` then inline it here; that pattern is used a lot in our utilities.

Also - fill in the other fields ( other than `url` and `install_requires` ).